manuscripts2/README.md

```diff
@@ -1,20 +1,123 @@
+The enriched data will be stored in the `github_prs` index.
+
+The git index has no changes and can be produced as follows:
+```bash
+p2o.py --enrich --index git_raw --index-enrich git -e http://localhost:9200 --no_inc --debug --db-host localhost --db-sortinghat <db name> --db-user root git <repository url>
+```
+
+**NOTE:** If you don't know how or what [Sortinghat](https://github.com/chaoss/grimoirelab-sortinghat) is, please [go through the tutorial](https://grimoirelab.gitbooks.io/tutorial/sortinghat/intro.html). Sortinghat helps us manage the identities of the contributors and manitainers.
+
+
+This is how you can use `manuscripts2` to generate the reports:
+
+```bash
+usage: manuscripts2 [-h] [-v] [-d DATA_DIR] [-e END_DATE] [-g] [-i INTERVAL]
+                    [-s START_DATE] [-u ELASTIC_URL]
+                    [--data-sources [DATA_SOURCES [DATA_SOURCES ...]]]
+                    [-n [NAME]] [--indices [INDICES [INDICES ...]]] [-l LOGO]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -v, --version         Show version
+  -d DATA_DIR, --data-dir DATA_DIR
+                        Directory to store the data results (default:
+                        GENERATED-REPORT')
+  -e END_DATE, --end-date END_DATE
+                        End date for the report (UTC) (<) (default: now)
+  -g, --debug
+  -i INTERVAL, --interval INTERVAL
+                        Analysis interval (month (default), quarter, year)
+  -s START_DATE, --start-date START_DATE
+                        Start date for the report (UTC) (>=) (default: None)
+  -u ELASTIC_URL, --elastic-url ELASTIC_URL
+                        Elastic URL with the enriched indexes
+  --data-sources [DATA_SOURCES [DATA_SOURCES ...]]
+                        Data source for the report (git, ...)
+  -n [NAME], --name [NAME]
+                        Report name (default: UnnamedReport)
+  --indices [INDICES [INDICES ...]]
+                        Indices to be used to generate the report (git_index,
+                        github_index ...)
+  -l LOGO, --logo LOGO  Provide a logo for the report. Allowed formats: .png,
+                        .pdf, .jpg, .mps, .jpeg, .jbig2, .jb2, .PNG, .PDF,
+                        .JPG, .JPEG, .JBIG2, .JB2, .eps
+```
+
+First, run:
+
+```bash
+python setup.py install
+```
+from the grimoirelab-manuscripts to configure the scripts.
+
+Now, once the indices mentioned above have been created, you can run the following command to generate the report:
+```bash
+manuscripts2 -n <name of the report/repository being analysed> -d <name of the directory to store the data in> -s <start date from which the repo has to be analysed> -e <end date upto which the repo is to be analysed> -i <time interval> -u <elasticsearch url where all the indices are being stored> --data-sources <the data sources from which the data is to be gathered from> --indices <custom index names to get the data from> -l <custom logo to be applied to the report>
```

Comment:
It's better if you use shorter names for the labels, such as `<repo_name>`. In a following paragraph, you can make it clear that `<repo_name>` is the "name of the project/repository being analysed" (note project instead of report). But also include an example (for example in this case, "CHAOSS").

Reply:
Explain also which data sources are possible (with the correct labels), and how the correspond, one by one, with the indexes.